app/main.py

```diff
@@ -1,3 +1,91 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float, int]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[Distance, float, int]) -> Distance:
```

Reply:
please fix everywhere

Reply:
> пожалуйста, исправьте везде

throws an error(

Reply:
If you add `from __future__ import annotations` it will work

Reply:
If you add from __future__ import annotations it will work